smelter-api/src/input/v4l2.rs

```diff
@@ -19,15 +19,18 @@ pub struct V4l2Input {
     /// detection order, the `by-id` paths are always the same for a given device, and the
     /// `by-path` paths should be the same for specific ports.
     pub path: Arc<Path>,
-    /// The resolution that will be negotiated with the device.
-    pub resolution: Resolution,
     /// The format that will be negotiated with the device.
     pub format: V4l2InputFormat,
+    /// The resolution that will be negotiated with the device.
```

Comment:
```suggestion
    /// The requested resolution that will be negotiated with the device.
```
or maybe desired 

normally I would think about changing the field name too, but since we released it I don't think it is worth it